src/app/CommandSender.cpp

```diff
@@ -330,5 +355,62 @@ TLV::TLVWriter * CommandSender::GetCommandDataIBTLVWriter()
     }
 }
 
+CHIP_ERROR CommandSender::SendTimedRequest(uint16_t aTimeoutMs)
+{
+    using namespace Protocols::InteractionModel;
+    using namespace Messaging;
+
+    // The payload is an anonymous struct (2 bytes) containing a single
+    // 16-bit integer with a context tag (1 control byte, 1 byte tag, at
+    // most 2 bytes for the integer).  Use MessagePacketBuffer::New to
+    // account for other message-global overheads (MIC, etc).
+    System::PacketBufferHandle payload = MessagePacketBuffer::New(6);
```

Comment:
1 control byte, 1 byte tag, at // most 2 bytes for the integer) == 4 bytes, then message-global overheads (MIC, etc) 2 bytes? Perhaps to declare one macro for TimedRequest payload, it would also be used for WriteClient

Reply:
> 1 control byte, 1 byte tag, at // most 2 bytes for the integer) == 4 bytes

Yes. Plus the two bytes for the struct (see immediately previous line of the comment), gives 6 bytes.

> then message-global overheads (MIC, etc) 

Is handled by the `MessagePacketBuffer` class.